docs/components/Combobox/Web.stories.tsx

```diff
@@ -94,8 +98,70 @@ const ComboboxChip: ComponentStory<typeof Combobox> = args => {
   );
 };
 
+const ComboboxClearSelection: ComponentStory<typeof Combobox> = args => {
+  const [selected, setSelected] = useState<ComboboxOption | null>(null);
```

Comment:
what do you think about having an initial option here so that you can immediately click the button to clear it rather than making a selection and then clearing?

Reply:
ya good call I'll update that